conda_forge_tick/update_recipe/version.py

```diff
@@ -897,7 +925,10 @@ def update_version(raw_meta_yaml, version, hash_type="sha256") -> (str, set[str]
     import argparse
 
     # set log level to debug
-    logging.basicConfig(level=logging.DEBUG)
+    from conda_forge_tick.utils import setup_logging
+
+    setup_logging("INFO")
```

Comment:
didn't you want to set the log level to debug?

Reply:
I think info is fine, too. I left the string there so people can change it easily.

Reply:
ah, then you should adjust the comment above

Reply:
Done!